flow-typed/redux.js

```diff
@@ -0,0 +1,74 @@
+declare module 'redux' {
+
+  /*
+
+    S = State
+    A = Action
+
+  */
+
+  declare type Dispatch<A: { type: $Subtype<string> }> = (action: A) => A;
```

Comment:
Bacause otherwise you get the following error (todos-flow example)

```
types/index.js:28
 28:     { type: 'ADD_TODO', id: Id, text: Text }
                 ^^^^^^^^^^ string literal `ADD_TODO`. Expected string literal `ADD_TODO`
 10:   declare type Dispatch<A: { type: string }> = (action: A) => A;
                                        ^^^^^^ string. See lib: ../../flow-typed/redux.js:10

types/index.js:29
 29:   | { type: 'TOGGLE_TODO', id: Id }
                 ^^^^^^^^^^^^^ string literal `TOGGLE_TODO`. Expected string literal `TOGGLE_TODO`
 10:   declare type Dispatch<A: { type: string }> = (action: A) => A;
                                        ^^^^^^ string. See lib: ../../flow-typed/redux.js:10

types/index.js:30
 30:   | { type: 'SET_VISIBILITY_FILTER', filter: VisibilityFilter }
                 ^^^^^^^^^^^^^^^^^^^^^^^ string literal `SET_VISIBILITY_FILTER`. Expected string literal `SET_VISIBILITY_FILTER`
 10:   declare type Dispatch<A: { type: string }> = (action: A) => A;
                                        ^^^^^^ string. See lib: ../../flow-typed/redux.js:10
```